### PR TITLE
Improve accessibility on login page

### LINKED ISSUE #3133: Accessibility on login page & Password Manager Support
Without going in the depth of WCAG & co, it would be great if we could have a little better accessibility, especially on the login page.

While it looks nice, the login/password fields are defined as "paper input 1" and "paper input 2", which is a big no-no for accessibility, and in my case, prevents my password manager (LastPass, but I suspect it would be the same for others) to understand it is a login form and where it should populate the login/password, resulting in having to manually copy/paste the password each time.

### PR DESCRIPTION
**Pull request by rianadon**
## Proposed change

- The Home Assistant logo is hidden to screen readers
- The alternate login methods can be navigated via screen reader & are identified as buttons

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #3133

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-form/ha-form-select.ts [hunk 1]
```diff
@@ -31,7 +33,12 @@ export class HaFormSelect extends LitElement implements HaFormElement {
 
   protected render(): TemplateResult {
     return html`
-      <paper-menu-button horizontal-align="right" vertical-offset="8">
+      <paper-menu-button
+        horizontal-align="right"
+        vertical-offset="8"
+        .opened="${this._opened}"
+        @keydown="${this._keyPressed}"
```
Comment (zsarnett) [inline src/components/ha-form/ha-form-select.ts:40]:
```suggestion
        .opened=${this._opened}
        @keydown=${this._keyPressed}
```

**Comment by rianadon** [comment]
I've updated with upstream; at this point it's just a few aria labels so that components are correctly handled by screen readers and tab key.

File: src/html/authorize.html.template [hunk 2]
```diff
@@ -33,7 +33,7 @@
   <body>
     <div class="content">
       <div class="header">
-        <img src="/static/icons/favicon-192x192.png" height="52" />
+        <img src="/static/icons/favicon-192x192.png" height="52" aria-hidden="true" />
```
Comment (bramkragten) [inline src/html/authorize.html.template:36]: Why would this be hidden?

Comment (rianadon) [reply src/html/authorize.html.template:36]: The image is right next to the "Home Assistant" header text, so giving the image an alt text of "Home Assistant" or "Home Assistant logo" I think would be repetitive. However, in retrospect I realize giving the image a null alt-text is the most common way [1] of handling this case, so I'm going to amend the PR.

[1] https://www.w3.org/WAI/tutorials/images/decorative/

**Review by bramkragten — APPROVED** [review]

**Pull request merged**